### DISCUSSION TITLE
Possibility to create elements smaller than 1 pixel: 0.5 px, 0.2 px, etc.

### DISCUSSION THREAD

**Discussion by piotrtrkostrzewski94** [Orama-Interactive/Pixelorama#984] [General]
Good evening.
Is it possible to add an option to create elements smaller than 1 pixel: 0.5 px, 0.2 px, etc.?  Sometimes this would be a useful option for creating smaller items. 
 Thank you very much for your time. 
 Have a good night.

**Comment by OverloadedOrama** [comment]
You mean drawing elements smaller than one pixel? This is impossible as the pixel is by definition the smallest part of a digital image. What you can do to replicate this effect is to simply work with larger canvases and large brush sizes, and use 1px brush sizes for the smaller elements.

**Reply by piotrtrkostrzewski94** [reply]
Yes, that's what I mean.
I guessed but I wanted to make sure.
Thank you very much.  However, I think I will try the line in combination with the pencil.
I used brushes once, but it wasn't enough to use their power.
